examples/nifgen_standard_function/measurement.py

```diff
@@ -163,6 +168,11 @@ def cancel_callback():
 def _create_nifgen_session(
     session_info: nims.session_management.SessionInformation,
 ) -> nifgen.Session:
+    options = {}
+    if service_options.use_simulation:
+        options["simulate"] = True
+        options["driver_setup"] = {"Model": "5423 (2CH)"}
+
     session_kwargs = {}
     if service_options.use_grpc_device:
         session_grpc_address = service_options.grpc_device_address
```

Comment:
FYI @Tharun-Sundar @dixonjoel If we extract a shared helper function for this duplicated GrpcSessionOptions code, the rest of this function is a lot more manageable.